app/domain/authentication/authn_jwt/identity_from_url_provider.rb

```diff
@@ -0,0 +1,18 @@
+module Authentication
+  module AuthnJwt
+    # Provides jwt identity from information in the URL
+    class IdentityFromUrlProvider < IdentityProviderInterface
+      def initialize(authentication_parameters)
+        @authentication_parameters = authentication_parameters
+      end
+
+      def provide_jwt_identity
```

Comment:
What do you think, in order to be on the safe side to change it to:
```
      def provide_jwt_identity
         Raise error unless self.identity_available?
        @authentication_parameters.username
      end
```

Reply:
If identity_availible? is false so 100% the `fetch_token_field_name` raises error because if its false there is no configured variable and if there is not configured variable error will be raised. So i don't think we need this change

Reply:
Done